nemo.yaml

```diff
@@ -0,0 +1,130 @@
+package:
+  name: nemo
+  version: 1.22.0
+  epoch: 0
+  description: conversational AI toolkit built for researchers working on ASR, TTS, LLMs, and NLP.
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    - x86_64
+  options:
+    no-depends: true
+  dependencies:
+    runtime:
+      - libgomp
+      - pytorch-cuda12
+      # These are to be run time dependencies and built as seperate wolfi packages?
+      # pytorch-cuda12 - Setting PATH necessary?
+      # k2
+      # graphviz
+      # ais cli
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - ffmpeg-dev
+      - libsndfile-dev
+      - libavdevice60
+      - freetype-dev
+      - swig
+      - py3.11-installer
+      - py3.11-pip
+      - python-3.11
+      - python-3.11-dev
+      - samurai
+      - # wolfi equivalent of sox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/NVIDIA/NeMo.git
+      tag: v${{package.version}}
+      expected-commit: 0b7467ebd158ef6876dd147189517b25ef626982
+
+  - runs: git submodule update --init --recursive
+
+  - runs: |
+      sed -i '/fasttext/d' requirements/requirements_nlp.txt
+      sed -i 's/megatron_core==0.4.0/megatron_core==0.3.0/' requirements/requirements_nlp.txt
+
+  - runs: |
+      # Setup the virtualenv
+
+      python -m venv .venv --system-site-packages
+
+      .venv/bin/pip install --upgrade wheel pip==23.3.2 setuptools==65.5.1
+
+      .venv/bin/pip install Cython
+      
+
+  - runs:
+      # install megatron-core from checkout specified in Dockerfile
+      git clone https://github.com/NVIDIA/Megatron-LM.git && \
+      cd Megatron-LM && \
+      git checkout 27cbe46714a50c43ed290f1b1472db8d2780c55c && \
+      .venv/bin/pip install .
+
+      # install apex(intended to address a bug in Nvidia's pytorch-23.11 container)
+      # But we are neither using this container as the base, nor this version of pytorch, the need to use this must be evaluated.
+          # this needs cuda drivers
+          # should this package move to extra-packages repository for the cuda packages?
+
+      # Install transformer engine
+          # same as Apex
+
+      # pip3 uninstall -y sacrebleu torchtext most likely not needed as these came from nvidia-pytorch image
+
+      # export LHOTSE_REQUIRE_TORCHAUDIO=0
+
+      # install requirements from all requirements_xyz.txt
+      for f in ./requirements/requirements*.txt;
+        do pip3 install --disable-pip-version-check --no-cache-dir -r $f
```

Comment:
this is actually done by setup.py below when '.[all]' is used.  the setup builds its requirements from "all" of the requirements/*.txt